README.md

```diff
@@ -28,10 +28,20 @@ In this project you will create a login page and request a token from the server
 
 Demonstrate your understanding of this Sprint's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
-- [ ] Explain what a token is used for.
-- [ ] What steps can you take in your web apps to keep your data secure?
-- [ ] Describe how web servers work.
-- [ ] Which HTTP methods can be mapped to the CRUD acronym that we use when interfacing with APIs/Servers.
+- [X] Explain what a token is used for.
+Tokens are like passwords that are unique to the user that gives them access to the server.
+
+- [X] What steps can you take in your web apps to keep your data secure?
+You can secure it bit better by building it with authentication and using secure and private routes.
```

Comment:
Hey Juan I wanted to to say I think the answers here could be a bit more in depth. Because currently I see a lot of student not giving really good answers. Reason why is because they think of it like college. Think of these question like interview questions. You may want to say something like - Most of the things that are needed to take secure data is: Testing often as possible for bugs and anything that could lead to potential data breaches, use of JWT Tokens they are issued by the server and are strings of cryptic text which then can be stored in the local storage or session storage on the client side. Uses of protected routes should only render with the authentication. This ensures safety of the users data by not displaying any information until a user has logged in.